### PR TITLE
DS-2066-pipeline-versions initial draft of pipeline versions docs

### PR DESCRIPTION
**Pull request by chtyler**
## Description
These changes contain documentation on data science pipeline versioning, to be available at RHOAI 2.7. 

## How Has This Been Tested?
These documentation changes have been tested against a build provided by the development team.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/viewing-pipeline-versions.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and contains a pipeline server.
+* You have a pipeline available on an active and available pipeline server.
+
+.Procedure
+. From the {productname-short} dashboard, click *Data Science Pipelines* -> *Pipelines*.
++
+The *Pipelines* page opens.
+. From the *Project* list, select the relevant project that contains the pipeline versions you want to view.
```
Comment (grainnejenningsRH) [inline modules/viewing-pipeline-versions.adoc:25]: 'select the project containing the pipeline versions that you want to view.'

File: modules/viewing-pipeline-versions.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+. Study the pipelines on the list.
```
Comment (grainnejenningsRH) [inline modules/viewing-pipeline-versions.adoc:26]: Could remove this line.

File: modules/viewing-pipeline-versions.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+. Click *Expand* (image:images/rhoai-expand-icon.png[]) on the row containing the relevant pipeline.
```
Comment (grainnejenningsRH) [inline modules/viewing-pipeline-versions.adoc:27]: Suggest updating to 'the row containing the pipeline that you want to view versions for.'

File: modules/viewing-the-details-of-a-pipeline-version.adoc [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and contains a pipeline server.
+* You have a pipeline available on an active and available pipeline server.
+
+.Procedure
+. From the {productname-short} dashboard, click *Data Science Pipelines* -> *Pipelines*.
++
+The *Pipelines* page opens.
+. From the *Project* list, select the relevant project that contains the pipeline versions you want to view the details of.
```
Comment (grainnejenningsRH) [inline modules/viewing-the-details-of-a-pipeline-version.adoc:25]: 'select the project containing the pipeline versions that you want to view details for.'

File: modules/viewing-the-details-of-a-pipeline-version.adoc [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+. Study the pipelines on the list.
```
Comment (grainnejenningsRH) [inline modules/viewing-the-details-of-a-pipeline-version.adoc:26]: Can remove this line.

File: modules/viewing-the-details-of-a-pipeline-version.adoc [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+. Click *Expand* (image:images/rhoai-expand-icon.png[]) on the row containing the relevant pipeline.
```
Comment (grainnejenningsRH) [inline modules/viewing-the-details-of-a-pipeline-version.adoc:27]: Suggest updating to 'the row containing the pipeline that you want to view versions for.'

File: modules/viewing-the-details-of-a-pipeline-version.adoc [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+. Click the pipeline version that you want to view the details of. 
++ 
+The *Pipeline details* page opens displaying the *Graph* tab.
```
Comment (grainnejenningsRH) [inline modules/viewing-the-details-of-a-pipeline-version.adoc:30]: 'The *Pipeline details* page opens, displaying the *Graph* and *YAML* tabs.'

Suggest adding comma and referring to second available tab.

File: modules/viewing-triggered-pipeline-runs.adoc [hunk 8]
```diff
@@ -33,7 +33,7 @@ The *Run details* page opens.
 A table opens that shows list of triggered runs. After a run has completed its execution, the run's status is displayed in the *Status* column in the table, indicating whether the run has succeeded or failed.
 
 .Verification
-* A list of previously triggered runs are displayed in the *Triggered* tab on the *Runs* page.
+* A list of previously triggered runs display in the *Triggered* tab on the *Runs* page.
```
Comment (grainnejenningsRH) [inline modules/viewing-triggered-pipeline-runs.adoc:36]: Going by IBM style, looks like this should be 'are displayed'.

File: modules/viewing-the-details-of-a-pipeline-version.adoc [hunk 9]
```diff
@@ -0,0 +1,36 @@
+:_module-type: PROCEDURE
+
+[id="viewing-the-details-of-a-pipeline-version_{context}"]
+= Viewing the details of a pipeline version
```
Comment (grainnejenningsRH) [inline modules/viewing-the-details-of-a-pipeline-version.adoc:4]: See previous comment RE: potentially combining this with modules/viewing-pipeline-versions.adoc.

**Review by grainnejenningsRH — CHANGES_REQUESTED** [review]

File: modules/viewing-the-details-of-a-pipeline-version.adoc (hunk 9 above)
Comment (chtyler) [reply modules/viewing-the-details-of-a-pipeline-version.adoc:4]: In general, we have used separate modules for things like this as there are different screens involved, each screen showing different components.

File: modules/deleting-a-pipeline-server.adoc [hunk 10]
```diff
@@ -4,7 +4,7 @@
 = Deleting a pipeline server
 
 [role='_abstract']
-After you have finished running your data science pipelines, you can delete the pipeline server. Deleting a pipeline server automatically deletes all of its associated pipelines and runs. If your pipeline data is stored in a database, the database is also deleted along with its meta-data. In addition, after deleting a pipeline server, you cannot create new pipelines or pipeline runs until you create another pipeline server.
+After you have finished running your data science pipelines, you can delete the pipeline server. Deleting a pipeline server automatically deletes all of its associated pipelines, pipeline versions and runs. If your pipeline data is stored in a database, the database is also deleted along with its meta-data. In addition, after deleting a pipeline server, you cannot create new pipelines or pipeline runs until you create another pipeline server.
```
[the remark this answers is not in the document]
Comment (chtyler) [reply modules/deleting-a-pipeline-server.adoc:7]: Fixed.

File: modules/duplicating-a-scheduled-pipeline-run.adoc [hunk 11]
```diff
@@ -17,33 +17,34 @@ ifdef::upstream[]
...
-The *Clone* page opens.
-. From the *Project* list, select the project that contains the pipeline whose run that you want to clone.
-. In the *Name* field, enter a name for the run that you want to clone.
-. In the *Description* field, enter a description for the run that you want to clone.
-. From the *Pipeline* list, select the pipeline containing the run that you want to clone.
-. To configure the run type for the run that you are cloning, in the *Run type* section, perform one of the following sets of actions:
-* Select *Run once immediately after create* to specify the run that you are cloning executes once, and immediately after its creation. If you selected this option, skip to step 10.
-* Select *Schedule recurring run* to schedule the run that you are cloning to recur.
+The *Duplicate* page opens.
+. From the *Project* list, select the project that contains the pipeline whose run that you want to duplicate.
```
[the remark this answers is not in the document]
Comment (chtyler) [reply modules/duplicating-a-scheduled-pipeline-run.adoc:29]: Fixed.

File: modules/overview-of-pipeline-versions.adoc [hunk 12]
```diff
@@ -0,0 +1,11 @@
+:_module-type: CONCEPT
+
+[id='overview-of-pipeline-versions_{context}']
+= Overview of pipeline versions
+
+[role='_abstract']
+You can manage incremental changes to pipelines in {productname-short} by using versioning. This allows you to develop and deploy pipelines iteratively, preserving a record of your changes. You can track and manage your changes on the {productname-short} dashboard allowing you to schedule and execute runs against all available versions of your pipeline.
```
[the remark this answers is not in the document]
Comment (chtyler) [reply modules/overview-of-pipeline-versions.adoc:7]: Fixed.

File: modules/viewing-pipeline-versions.adoc (hunk 2 above)
Comment (chtyler) [reply modules/viewing-pipeline-versions.adoc:26]: Fixed.

File: modules/viewing-the-details-of-a-pipeline-version.adoc (hunk 5 above)
Comment (chtyler) [reply modules/viewing-the-details-of-a-pipeline-version.adoc:26]: Fixed.

File: modules/viewing-the-details-of-a-pipeline-version.adoc (hunk 6 above)
Comment (chtyler) [reply modules/viewing-the-details-of-a-pipeline-version.adoc:27]: Fixed.

File: modules/viewing-pipeline-versions.adoc (hunk 3 above)
Comment (chtyler) [reply modules/viewing-pipeline-versions.adoc:27]: Fixed.

File: modules/viewing-pipeline-versions.adoc (hunk 1 above)
Comment (chtyler) [reply modules/viewing-pipeline-versions.adoc:25]: Fixed.

File: modules/viewing-the-details-of-a-pipeline-version.adoc (hunk 4 above)
Comment (chtyler) [reply modules/viewing-the-details-of-a-pipeline-version.adoc:25]: Fixed.

File: modules/viewing-the-details-of-a-pipeline-version.adoc (hunk 7 above)
Comment (chtyler) [reply modules/viewing-the-details-of-a-pipeline-version.adoc:30]: Fixed.

File: modules/viewing-triggered-pipeline-runs.adoc (hunk 8 above)
Comment (chtyler) [reply modules/viewing-triggered-pipeline-runs.adoc:36]: Fixed.

File: modules/viewing-pipeline-versions.adoc [hunk 13]
```diff
@@ -0,0 +1,33 @@
+:_module-type: PROCEDURE
+
+[id="viewing-pipeline-versions_{context}"]
+= Viewing pipeline versions
+
+[role='_abstract']
+You can view the details of pipeline versions that you have uploaded to {productname-long}, such as its creation date and its runs.
```
[the remark this answers is not in the document]
Comment (chtyler) [reply modules/viewing-pipeline-versions.adoc:7]: In general, we have used separate modules for things like this as there are different screens involved, each screen showing different components.

[the remark this answers is not in the document]
Comment (chtyler) [reply modules/viewing-pipeline-versions.adoc:7]: Using your suggestion on making it distinct.

**Review by grainnejenningsRH — APPROVED** [review]

**Comment by chtyler** [comment]
@jgarciao - could you please review, thanks.

**Review by jgarciao — APPROVED** [review]
Add a minor correction but so far it looks good to me

File: modules/deleting-a-pipeline-version.adoc [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and contains a pipeline server.
+* You have imported a pipeline to an active and available pipeline server.
+* You have previously triggered a pipeline run.
```
[the remark this answers is not in the document]
Comment (chtyler) [reply modules/deleting-a-pipeline-version.adoc:20]: Removed the prereq.

**Pull request merged**